src/Symfony/Bundle/TwigBundle/CacheWarmer/TemplateCacheCacheWarmer.php

```diff
@@ -1,123 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Bundle\TwigBundle\CacheWarmer;
-
-use Psr\Container\ContainerInterface;
-use Symfony\Bundle\FrameworkBundle\CacheWarmer\TemplateFinderInterface;
-use Symfony\Component\DependencyInjection\ServiceSubscriberInterface;
-use Symfony\Component\Finder\Finder;
-use Symfony\Component\HttpKernel\CacheWarmer\CacheWarmerInterface;
-use Twig\Environment;
-use Twig\Error\Error;
-
-/**
- * Generates the Twig cache for all templates.
- *
- * This warmer must be registered after TemplatePathsCacheWarmer,
- * as the Twig loader will need the cache generated by it.
- *
- * @author Fabien Potencier <fabien@symfony.com>
- */
-class TemplateCacheCacheWarmer implements CacheWarmerInterface, ServiceSubscriberInterface
```

Comment:
do we not need this class anymore?

Reply:
`TemplateCacheWarmer` already does the job using the `twig.template_iterator`; `TemplateCacheCacheWarmer` is part of the legacy layer when `TwigEngine` is enabled under `templating` option.

Reply:
Note that `TemplatePathsCacheWarmer` was deprecated and removed too.